frontend/js/tests/stats/UserEntityChart.test.tsx

```diff
@@ -28,14 +28,8 @@ const userProps = {
     id: 0,
     name: "dummyUser",
   },
-  apiUrl: "apiUrl",
-  newAlert: jest.fn(),
 };
 
-const sitewideProps = {
-  apiUrl: "apiUrl",
```

Comment:
Same as above we want to keep this line

Reply:
OK we don't want to keep the apiUrl line, but we do want to keep an empty object in case props change in the future: `const sitewideProps = {};`